src/App.css

```diff
@@ -116,41 +118,37 @@ h1 {
   text-align: left;
 }
 .jobcard-title {
-  margin-bottom: 5px;
   font-size: 22px;
   font-family: "Roboto", sans-serif;
-  line-height: 28px;
   font-weight: 500;
+  line-height: 28px;
+  margin-bottom: 5px;
 }
-.jobcard-benefit {
-  padding: 0 0 0 19px;
-  margin: 10px 0px;
+.benefit-list {
   font-weight: 500;
   font-size: 14px;
+  margin: 10px 0px;
+  padding: 0 0 0 19px;
```

Comment:
why not `padding-left 19px`?

Reply:
yeah i change this part  to padding-left: 19px